azure-iot-hub-devicesdk/samples/send_to_output_module.py

```diff
@@ -0,0 +1,50 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import os
+import time
+from azure.iot.hub.devicesdk.module_client import ModuleClient
+from azure.iot.hub.devicesdk.auth.authentication_provider_factory import from_connection_string
+from azure.iot.hub.devicesdk.message import Message
+import uuid
+
+# The connection string for a device should never be stored in code. For the sake of simplicity we're using an environment variable here.
+conn_str = os.getenv("IOTHUB_MODULE_CONNECTION_STRING")
+# The "Authentication Provider" is the object in charge of creating authentication "tokens" for the device client.
+auth_provider = from_connection_string(conn_str)
+# For now, the SDK only supports MQTT as a protocol. the client object is used to interact with your Azure IoT hub.
+# It needs an Authentication Provider to secure the communication with the hub, using either tokens or x509 certificates
```

Comment:
This sample does not interact with IoTHub.  It interacts with IoTEdge and it only works with IotEdgeAuthenticationProvider, though I'm not sure how deeply you want to explain that here.

Reply:
that's right. suggestion:
"Inputs/Ouputs are only supported with Azure IoT Edge and the module client"

---
In reply to: [246254837](https://github.com/Azure/azure-iot-sdk-python-preview/pull/35#discussion_r246254837) [](ancestors = 246254837)